python_modules/libraries/dagster-aws/dagster_aws/ecs/launcher.py

```diff
@@ -429,39 +432,16 @@ def _get_run_tags(self, run_id: str) -> Tags:
     def _get_command_args(self, run_args: ExecuteRunArgs, context: LaunchRunContext):
         return run_args.get_command_args()
 
-    def _get_image_for_run(self, context: LaunchRunContext) -> Optional[str]:
-        job_origin = check.not_none(context.job_code_origin)
-        return job_origin.repository_origin.container_image
+    @staticmethod
+    def get_image_for_run(run: DagsterRun) -> Optional[str]:
+        return (
+            run.tags.get("dagster/image") or run.job_code_origin.repository_origin.container_image
```

Comment:
@gibsondan drive-by: it makes sense to check `dagster/image` for a manual override right? 

I'm also setting it in tests

Reply:
we decided to remove this logic